graylog2-server/src/main/java/org/graylog2/system/stats/ClusterStatsService.java

```diff
@@ -53,8 +51,8 @@
     private final AlertService alertService;
     private final AlarmCallbackConfigurationService alarmCallbackConfigurationService;
     private final DashboardService dashboardService;
+    private final Cluster cluster;
     private final IndexSetRegistry indexSetRegistry;
```

Comment:
After extracting `elasticsearchStats` to `Cluster` we can now get rid of `IndexSetRegistry` here.

Reply:
:heavy_check_mark: